qiita_pet/handlers/study_handlers/tests/test_artifact.py

```diff
@@ -140,6 +140,33 @@ def test_delete_artifact(self):
         wait_for_prep_information_job(1)
 
 
+class ArtifactGetSamplesTest(TestHandlerBase):
+    def test_get(self):
+        response = self.get('/artifact/samples/', {'ids[]': [4, 5]})
+        self.assertEqual(response.code, 200)
+        exp = (
+            '{"status": "success", "msg": "", "data": {"4": ['
```

Comment:
This test might fail if the order of the samples or of the keys changes, a better way to test this is to load the JSON string as a Python dictionary and assert on that object with `asserEqual`.